src/drawer.js

```diff
@@ -166,6 +166,38 @@ $.Drawer.prototype = {
         return this;
     },
 
+    /**
+     * This function gets the top left point from the viewport using the pixel point
+     * @param {OpenSeadragon.Point} point - the pixel points to convert
+     */
+    viewportCoordToDrawerCoord: function(point) {
+        var topLeft = this.viewport.pixelFromPointNoRotate(point, true);
```

Comment:
Why NoRotate here?

Reply:
Logic is based on point provided at 0 degree, thus we don't need to worry about rotation after cropping.